app/assets/stylesheets/components/about.scss

```diff
@@ -134,13 +134,13 @@
   .contribute {
     color: #999;
     background: linear-gradient(white, $light-grey);
-
+    margin: 0;
     .cta {
         text-decoration: none;
         background-color: $button-primary;
         color: #ffffff;
```

Comment:
Nit.

You didn't make this change but can you please replace the hardcoded color with the `$white` variable defined in `settings.scss`?